### PR TITLE
Fix docstring dataset get aux data definition

### PR DESCRIPTION
**Pull request by HansN87**
should fix https://github.com/IceCubeOpenSource/skyllh/issues/26

### CODE REVIEW

File: skyllh/core/dataset.py [hunk 1]
```diff
@@ -1043,30 +1043,28 @@ def add_aux_data_definition(self, name, pathfilenames):
...
-            If auxiliary data is already stored under the given name.
-
+            If auxiliary data with the given name does not exist.
 
         Returns
         -------
-        _aux_data[name] : list of strings
-            The locations of the files defined in the auxiliary data
+        _aux_data[name] : list of str
+            Locations (pathfilenames) of the files defined in the auxiliary data \
```
[the remark this answers is not in the document]
Comment (HansN87) [reply skyllh/core/dataset.py]: fix in https://github.com/IceCubeOpenSource/skyllh/pull/30/commits/4899492e1c3f33443738cf8f76c5911acb624e05

File: skyllh/core/dataset.py [hunk 2]
```diff
@@ -1043,30 +1043,28 @@ def add_aux_data_definition(self, name, pathfilenames):
...
         KeyError
-            If auxiliary data is already stored under the given name.
-
+            If auxiliary data with the given name does not exist.
 
         Returns
         -------
-        _aux_data[name] : list of strings
-            The locations of the files defined in the auxiliary data
+        _aux_data[name] : list of str
```
[the remark this answers is not in the document]
Comment (HansN87) [reply skyllh/core/dataset.py]: fix in https://github.com/IceCubeOpenSource/skyllh/pull/30/commits/4899492e1c3f33443738cf8f76c5911acb624e05

File: skyllh/core/dataset.py [hunk 3]
```diff
@@ -1057,8 +1057,8 @@ def get_aux_data_definition(self, name):
 
         Returns
         -------
-        _aux_data[name] : list of str
-            Locations (pathfilenames) of the files defined in the auxiliary data \
+        _aux_data_definition[name] : list of str
```
Comment (martwo) [inline skyllh/core/dataset.py]:
```suggestion
        aux_data_definition : list of str
```

File: skyllh/core/dataset.py [hunk 4]
```diff
@@ -1043,30 +1043,28 @@ def add_aux_data_definition(self, name, pathfilenames):
...
+        aux_data_definition : list of str
+            The locations (pathfilenames) of the files defined in the auxiliary data
+                    as auxiliary data definition.
         """
         
         if(not name in self._aux_data_definitions):
-            raise KeyError('The auxiliary data definition "%s" does not '
-                'exist in dataset "%s"!'%(name, self.name))
+            raise KeyError('The auxiliary data definition {} does not '
+                'exist in dataset {}!'.format(name, self.name))
```
Comment (martwo) [inline skyllh/core/dataset.py]:
```suggestion
                'exist in dataset "{}"!'.format(name, self.name))
```

File: skyllh/core/dataset.py [hunk 5]
```diff
@@ -1064,7 +1064,7 @@ def get_aux_data_definition(self, name):
         
         if(not name in self._aux_data_definitions):
             raise KeyError('The auxiliary data definition {} does not '
```
Comment (martwo) [inline skyllh/core/dataset.py]:
```suggestion
            raise KeyError('The auxiliary data definition "{}" does not '
```

**Pull request merged**